singlestoredb_cloud/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+SingleStoreDB Cloud is a distributed, relational database with optimized speed and scalability to support data-intensive and real-time applications. Using the Datadog integration, you can monitor your SingleStoreDB Cloud databases. To integrate Datadog with SingleStoreDB Cloud, install the Datadog SingleStore integration and then configure it in the [Cloud Portal][1]. 
+
```

Comment:
Updated overview section to include the value "Using this Integration, you can monitor overall health and performance of a SinglestoreDB Cloud Workspace Group/Cluster"

Reply:
Hi @tdasarathan , more specifically I think it would be useful if you mentioned:
- the types of health and performance data offered
- what this data looks like (will it be in metrics, logs, etc?)

Reply:
Updated, please review